src/app/search/search-filters/freeform.ts

```diff
@@ -150,17 +150,20 @@ const freeformFilters: FilterDefinition[] = [
         // favor items we actually own
         return Array.from(
           new Set([...myPerkNames, ...allPerkNames]),
-          (s) => `perkname:${quoteFilterString(s)}`,
+          (s) => `exactperk:${quoteFilterString(s)}`,
         );
       }
     },
-    filter: ({ filterValue, language, d2Definitions }) => {
-      const startWord = startWordRegexp(plainString(filterValue, language), language);
-      const test = (s: string) => startWord.test(plainString(s, language));
+    filter: ({ lhs, filterValue, language, d2Definitions }) => {
+      const normalized = plainString(filterValue, language);
+      let test = (s: string) => normalized === plainString(s, language);
+      if (lhs === 'perkname') {
+        test = (s: string) => normalized === plainString(s, language);
+      }
```

Comment:
These two predicates are identical, so `perkname` also gets the new behavior.

Reply:
Whoops, not sure how that happened